sdks/python/tox.ini

```diff
@@ -290,7 +290,7 @@ commands =
 [testenv:docs]
 extras = test,gcp,docs,interactive
 deps =
-  Sphinx==1.6.5
+  Sphinx==1.8.5
   sphinx_rtd_theme==0.2.4
```

Comment:
Is there a JIRA for updating this? (0.4.3 is the latest)

Reply:
Sorry, did you mean why update Sphinx and not update sphinx_rtd_theme?

Reply:
I meant that we should update sphinx_rtd_theme, but in a different PR.

Reply:
Created: https://issues.apache.org/jira/browse/BEAM-9163

Reply:
Got it. Thank you!